lib/api/1.1/northbound/tags.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    express = require('express'),
+    parser = require('body-parser');
+
+module.exports = tagsRouterFactory;
+
+di.annotate(tagsRouterFactory, new di.Provide('Http.Api.Tags'));
+di.annotate(tagsRouterFactory, new di.Inject(
+        'Http.Services.Api.Tags',
+        'Http.Services.Api.Nodes',
+        'Http.Services.RestApi',
+        '_',
+        'Promise'
+    )
+);
+
+function tagsRouterFactory (
+    Tags,
+    Nodes, 
+    rest, 
+    _, 
+    Promise
+) {
+    var router = express.Router();
+
+    router.get('/tags', rest(function (req) {
```

Comment:
the same with above,  need annotations, v1.1 api apidoc, and python-client is based on router's annotations.